### PR TITLE
may-newsletter-plus-more

### PR DESCRIPTION
**Pull request by anaisconce**

### CODE REVIEW

File: help/sharing.md [hunk 1]
```diff
@@ -83,6 +83,17 @@ i.e. when the Public Access role is set to "Viewer." Read more about
...
+
+Non-owners may look up their access details to a document by clicking on the share icon 
+(<span class="grist-icon" style="--icon: var(--icon-Share)"></span>) on top right of the
+screen and selecting "Access Details."
+
+![Access Details](../images/newsletters/2022-05/access-details.png)
+
+From the access details pop up, you may click the trash icon to leave a document.
+
+![Access Details](images/sharing/sharing-access-details.png)
```
Comment (dsagal) [inline help/sharing.md:99]: I think it's good to hide your email address from documentation, but the screenshot looks a little weird with an empty line in its space. Here's a screenshot with a fake email -- would that look better?
Screen Shot 2022-05-31 at 1 16 12 PM

File: help/keyboard-shortcuts.md [hunk 2]
```diff
@@ -59,6 +59,7 @@
 | - | - | - | 
 | <code class="keys">*⌘* *⇧* *=*</code> | <code class="keys">*Ctrl* + *Shift* + *=*</code> | Insert a new record, before the currently selected one in an unsorted table |
 | <code class="keys">*⌘* *=*</code> | <code class="keys">*Ctrl* + *=*</code> | Insert a new record, after the currently selected one in an unsorted table |
+| <code class="keys">*⌘* *⇧* *D*</code> | <code class="keys">*Ctrl* + *Shift* + *D*</code> | Duplicate the currently selected record(s) |
```
Comment (dsagal) [inline help/keyboard-shortcuts.md:62]: Thank you! 🙏

File: help/newsletters/2022-05.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+### Linking Referenced Data to Summary Tables
+
+Tables that reference a summary table’s underlying data may now be linked to the summary table itself. 
+
+<span class="screenshot-large">*![Linking Referenced Data to Summary Table](../images/newsletters/2022-05/summary-reference-linking.gif)*</span>
+
+### API Endpoint GET /attachments
+
+New API endpoint. /attachments will return list of all attachment metadata. [Learn more.](https://support.getgrist.com/api/#tag/attachments){:target="\_blank"}
```
Comment (dsagal) [inline help/newsletters/2022-05.md:155]: Ideally the link should have `(../api.md#tag/attachments)` here in the Help Site (but the full link in SendGrid)

File: help/newsletters/2022-05.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+    ![Raw Data Page](../images/newsletters/2022-05/raw-data-summary.png)
+
+</div>
+
+<div class="col-md-6" markdown="1">
+
+* **Link Related Data to Summary Tables**
+
+    Data that references the underlying data of a summary table may now reference the summary table itself.
```
Comment (dsagal) [inline help/newsletters/2022-05.md:118]: This is hard to parse. Maybe better to be more hand-wavy here, like "When linking summary tables and other widgets, you can now link on reference columns too"?

File: help/newsletters/2022-05.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+
+</div>
+
+</div>
+
+## What's New
+
+### Raw Data Tables
+
+Introducing the raw data page, a special page that lists all data tables in your document, making it easier for creators to find and work with their data outside of dashboards. [Learn more.](../raw-data.md){:target="\_blank"}
```
Comment (dsagal) [inline help/newsletters/2022-05.md:137]: Maybe better to end with a simpler message, like "making it easier for you to find and browse all your data"?

File: help/newsletters/2022-05.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+
+
+* **Next webinar -- Expense Tracking in Grist v. Excel**
+
+    How to build efficient workflows in Grist.
+
+* **2 new templates**
+
+    Hurricane preparedness and gig staffing.  
+
```
Comment (dsagal) [inline help/newsletters/2022-05.md:128]: Maybe could tighten up things a tiny bit and fit one more item for the "Shortcut to duplicate rows"?

**Review by dsagal — COMMENTED** [review]
Thank you!

File: help/newsletters/2022-05.md [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+Introducing the raw data page, a special page that lists all data tables in your document, making it easier for creators to find and work with their data outside of dashboards. [Learn more.](../raw-data.md){:target="\_blank"}
+
+Raw Data has been a long time in coming! In Grist, there has always been a conceptual separation between views and underlying data. As a Grist creator, you have probably found yourself in the shoes of an application developer: at one time thinking about relationships in the data, and separately designing how to present this data in convenient productive views.
+
+Raw data makes the separation explicit. It’s a place where you can always find the underlying data, and its structure.
+
+When creating pages, your page list can now be limited to only the most useful views, without the clutter of helper tables. Those are still available any time in the raw data view.
+
+![Raw Data](../images/newsletters/2022-05/raw-data.png)
```
[the remark this answers is not in the document]
Comment (anaisconce) [reply help/newsletters/2022-05.md:145]: I tried and it looked too compressed.

image

I resized the original image a bit and it looks better now. 👍

File: help/sharing.md [hunk 8]
```diff
@@ -83,6 +83,17 @@ i.e. when the Public Access role is set to "Viewer." Read more about
...
+Access Rules are supported with link sharing when Public Access
+is set to "Viewer" or "Editor."
+
+## Leaving a Document
+
+Non-owners may look up their access details to a document by clicking on the share icon 
+(<span class="grist-icon" style="--icon: var(--icon-Share)"></span>) on top right of the
+screen and selecting "Access Details."
+
+![Access Details](../images/newsletters/2022-05/access-details.png)
```
[the remark this answers is not in the document]
Comment (anaisconce) [reply help/sharing.md:95]: Same issue as last time, screen-shot half looked too compressed and blurry. Let me resize the original image again. I agree it's big.

File: help/sharing.md (hunk 1 above)
Comment (anaisconce) [reply help/sharing.md:99]: Thanks! Swapped it out.

**Review by dsagal — APPROVED** [review]
